spec/features/crops/crop_detail_page_spec.rb

```diff
@@ -4,6 +4,61 @@
 
   let(:crop) { FactoryGirl.create(:crop) }
 
+  context "varieties" do
+    let!(:roma1) { FactoryGirl.create(:crop, :name => 'Roma tomato 1', :parent => crop) }
+    let!(:roma2) { FactoryGirl.create(:crop, :name => 'Roma tomato 2', :parent => crop) }
+    let!(:roma3) { FactoryGirl.create(:crop, :name => 'Roma tomato 3', :parent => crop) }
+    let!(:roma4) { FactoryGirl.create(:crop, :name => 'Roma tomato 4', :parent => crop) }
+
+    scenario "The crop has 4 varieties" do
+
+      visit crop_path(crop)
+
+      # It lists all 5 items (note: including the top level item.)
+      # It DOES NOT have "Show all/less" toggle link
+      expect(page).to have_css('li', text: /tomato/i, count: 5)
+      expect(page).to have_no_css('button', text: /Show all+/i)
+      expect(page).to have_no_css('button', text: /Show less+/i)
```

Comment:
Thanks, done. I guess I was originally trying to match its classes as well so I thought I needed have_css, but decided to omit it. So you're right have_selector does the job.